### PR TITLE
Additional language on dialpad

### PR DESCRIPTION
**Pull request by Belyavin**
Fixes FossifyOrg/Phone#47

### CODE REVIEW

File: app/src/main/res/values/strings.xml
Comment (Aga-C) [inline app/src/main/res/values/strings.xml:28]: Instead of creating a new string, please use `none` from Commons.

**Comment by Aga-C** [comment]
Letters in both languages should be centered (e.g. 3 looks really bad).

image

**Comment by derrohrbach** [comment]
@Aga-C also the height of the lowest row of keys is different, which looks weird. 

And also # and * are not vertically aligned with the 0, but this is also the case on the master branch. Might be a different PR?

**Comment by Belyavin** [comment]
I've changed the string to a common one, and aligned the text to the center. I also aligned * and # to 0, though I'm not sure if it became better, need you opinion on this.

**Comment by Belyavin** [comment]
New text alignment

**Review by HenriDellal — CHANGES_REQUESTED** [review]

**Comment by Belyavin** [comment]
Hi, are we waiting for something before merging?

**Comment by Aga-C** [comment]
@Belyavin You have to wait for the final review from Naveen.

File: app/src/main/res/layout/activity_settings.xml
Comment (naveensingh) [inline app/src/main/res/layout/activity_settings.xml:278]: This is still referencing the deleted `none_explicit` string.

File: app/src/main/kotlin/org/fossify/phone/helpers/DialpadT9.kt
Comment (naveensingh) [inline app/src/main/kotlin/org/fossify/phone/helpers/DialpadT9.kt:65]: Useless `;`

File: app/src/main/kotlin/org/fossify/phone/helpers/DialpadT9.kt
Comment (naveensingh) [inline app/src/main/kotlin/org/fossify/phone/helpers/DialpadT9.kt:86]: The code in `DialpadT9.kt` isn't properly formatted in many places. 

https://stackoverflow.com/questions/16580171/code-formatting-shortcuts-in-android-studio-for-operation-systems

File: app/src/main/kotlin/org/fossify/phone/helpers/DialpadT9.kt
Comment (naveensingh) [inline app/src/main/kotlin/org/fossify/phone/helpers/DialpadT9.kt:27]: There is no space between `if` and `(`...

File: app/src/main/kotlin/org/fossify/phone/helpers/DialpadT9.kt
Comment (naveensingh) [inline app/src/main/kotlin/org/fossify/phone/helpers/DialpadT9.kt:45]: There is no space between `)` and `{`...

File: app/src/main/res/layout/dialpad.xml
Comment (naveensingh) [inline app/src/main/res/layout/dialpad.xml:60]: The attribute `android:layout_below` is not valid in the `LinearLayout`. It's there in every button from `dialpad_2_holder` to `dialpad_0_holder` except the asterisk.

File: app/src/main/res/layout/dialpad.xml
Comment (naveensingh) [inline app/src/main/res/layout/dialpad.xml:272]: `android:layout_centerInParent="true"` is not valid in the LinearLayout.

File: app/src/main/res/layout/dialpad.xml
Comment (naveensingh) [inline app/src/main/res/layout/dialpad.xml:322]: `android:layout_centerInParent="true"` is not valid in the LinearLayout.

File: app/src/main/res/layout/dialpad.xml
Comment (naveensingh) [inline app/src/main/res/layout/dialpad.xml:343]: `android:layout_centerInParent="true"` is not valid in the LinearLayout.

**Review by naveensingh — CHANGES_REQUESTED** [review]

**Comment by naveensingh** [comment]
> I also aligned * and # to 0, though I'm not sure if it became better, need you opinion on this.

@Belyavin indeed * and # look better when centered, I reverted that change.

**Pull request closed without merging**
